mitiq/_typing.py

```diff
@@ -45,4 +52,20 @@
 except ImportError:  # pragma: no cover
     _BKCircuit = _Circuit
 
+# Supported + installed quantum programs.
 QPROGRAM = Union[_Circuit, _Program, _QuantumCircuit, _BKCircuit]
+
+# Supported measurement results.
+Bitstring = List[int]
+MeasurementResult = List[Bitstring]
```

Comment:
This is a question not a suggesiton: 

What happens if an executor returns the measurement results as an `np.ndarray`. 
Do we want to explicitly support this case with  `MeasurementResult = Union[List[Bitstring], np.ndarray]` . 
The disadvantage is that `np.ndarray` is a very ambiguous type.

Reply:
It's possible, but we also want to use `np.ndarray` for wavefunctions and density matrices. I'm very close to suggesting we define data classes for everything an executor can return. This breaks the (very strict) rule of "no new objects a user wouldn't already be using," but I think it's very low overhead and would make our lives much easier. In that case `MeasurementResult` could contain `np.ndarray` as you suggest, it would just have to be wrapped as `mitiq.Bitstring(measurement_array)`.